.github/workflows/CI-BUILD.yml

```
@@ -184,34 +240,175 @@ jobs:
+      - name: Download All Artifacts
+        uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093 # v4.3.0
+        with:
+          path: ${{ github.workspace }}/dist
+          pattern: multicast-build-${{ github.sha }}
+          merge-multiple: true
+      - name: Calculate artifact digests
+        run: |
+          shasum -a 512 dist/* > build.checksums.txt
+      - name: "Attest Build Checksums"
+        id: multicast-build-chksum-attest
+        if: ${{ !cancelled() && (github.repository == 'reactive-firewall-org/multicast') && (needs.BUILD.envs.ENVIRONMENT == 'Deployment' && startsWith(github.ref, 'refs/tags/v')) }}
```

actionlint (1.7.7)

property "envs" is not defined in object type {outputs: {artifact-digest: string; artifact-files: string; artifact-id: string; artifact-url: string; build_environment: string; build_status: string}; result: string}

(expression)